src/test/java/racing/domain/CarManagerTest.java

```diff
@@ -0,0 +1,66 @@
+package racing.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class CarManagerTest {
+
+  @Test
+  void 자동차_객체_생성() {
+    String[] carNames = {"pobi", "crong", "honux"};
+    assertThat(new CarManager(carNames, new RandomGenerator()).getCars().size()).isEqualTo(3);
+
+  }
+
+  @Test
+  void 우승자_구하기() {
+    Car pobi = new Car("pobi", 3);
+    Car jason = new Car("jason", 1);
+    Car brown = new Car("brown", 2);
+
+    List<Car> cars = List.of(pobi, jason, brown);
+
+    CarManager carManager = new CarManager(cars);
+
+    List<String> winners = carManager.findWinners();
+
+    assertThat(winners).containsExactlyInAnyOrder("pobi");
+  }
+
+  @Test
+  void 경기후_결과_테스트_멈춤_케이스() {
+    Car pobi = new Car("pobi");
+    Car jason = new Car("jason");
+    Car brown = new Car("brown");
+
+    List<Car> cars = List.of(pobi, jason, brown);
+
+    CarManager carManager = new CarManager(cars, new FixedNumberGenerator(2));
```

Comment:
```suggestion
    CarManager carManager = new CarManager(cars, () -> 2);
```
만들어주신 `NumberGenerator`는 `@FunctionalInterface`이기 때문에 이런식으로 구현체 없이 람다식으로도 표현가능합니다 😄